src/styles/utils/variables.scss

```diff
@@ -0,0 +1,9 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
```

Comment:
It's better to upload the font from HTML. Also, could you please specify the needed font weights to pass the tests

Reply:
fonts has been uploaded